src/WebJobs.Extensions.DurableTask/DurableTaskWebJobsStartup.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+#if NETSTANDARD2_0
```

Comment:
nit: I wonder if we should move this `#if` up one line. It seems weird to compile just a `using` statement in the case of .NET Framework.